### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by NitBravoA92**
- Creation of the desktop version of the Headline section.
- Creation of the desktop version of the Works section.
- Creation of the desktop version of the About Myself section.
- Creation of the desktop version of the Contact form section.
- Change the background images of the sections.
- Implementation of the hover effect in the cards of the Works section.
- Adding new markup to show different information in the Works section cards for the desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -71,6 +71,14 @@ body {
   border-radius: 50%;
 }
 
+.desktop {
+  display: none !important;
```
Comment (zieeco) [inline style.css]: - Hey, let's find a better way of applying this rule in achieving what we want as overusing the `!important` rule is regarded as a bad practice in the microverse  [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), and as such, you shouldn't use it in your CSS rule except using a CSS framework, let' find a better way of applying the rule.

File: style.css [hunk 2]
```diff
@@ -71,6 +71,14 @@ body {
...
+}
+
+.mobile {
+  display: inline-block !important;
```
Comment (zieeco) [inline style.css]: - Hey, the same issue is applicable here as pointed out in my previous review.

File: style.css [hunk 3]
```diff
@@ -683,6 +684,14 @@ footer {
...
 
 @media only screen and (min-width: 768px) {
+  .desktop {
+    display: inline-block !important;
+  }
+
+  .mobile {
+    display: none !important;
+  }
+
```
Comment (zieeco) [inline style.css]: - Hey, the same issue is applicable here as pointed out in my previous review.

Comment (zieeco) [reply style.css]: - Hey, the same issue is applicable here as pointed out in my previous review in these [lines of code](https://github.com/NitBravoA92/portfolio-setup-and-mobile-first/blob/cf4b0431c838d994ae8ea09ffdcb65f89c297582/style.css#L74-L80) too.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @NitBravoA92,



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @NitBravoA92  , 
# STATUS: APPROVED! 🥇 💯 
- [x] Very Nice Design, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**